flair/datasets/treebanks.py

```diff
@@ -1303,3 +1306,30 @@ def __init__(self, base_path: Union[str, Path] = None, in_memory: bool = True):
         )
 
         super(UD_GOTHIC, self).__init__(data_folder, in_memory=in_memory)
+     
+    
+class UD_OLD_FRENCH(UniversalDependenciesCorpus):
+    def __init__(self, base_path: Union[str, Path] = None, in_memory: bool = True):
+
+        if type(base_path) == str:
+            base_path: Path = Path(base_path)
+
+        # this dataset name
+        dataset_name = self.__class__.__name__.lower()
+
+        # default dataset folder is the cache root
+        if not base_path:
+            base_path = Path(flair.cache_root) / "datasets"
+        data_folder = base_path / dataset_name
+
+        # download data if necessary
+        web_path = "https://github.com/UniversalDependencies/UD_Old_French-SRCMF/tree/master"
```

Comment:
This does not download the correct file. Please check the downloaded file if it looks correct.

Reply:
The urls are fixed with the new commits added.